platform-sdk/swirlds-base/src/test/java/com/swirlds/base/context/ThreadLocalContextTest.java

```diff
@@ -16,35 +16,44 @@
 
 package com.swirlds.base.context;
 
-import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-
+import com.swirlds.base.context.internal.GlobalContext;
 import com.swirlds.base.context.internal.ThreadLocalContext;
-import com.swirlds.base.test.fixtures.context.WithContext;
 import java.util.Map;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-@WithContext
```

Comment:
This test should use https://github.com/withcontext instead of the manual work in the setUp/tearDown method. in the maven project that was not possible based on missing textFixtures but here it should work